### PR TITLE
Add storage.md section to docs

### PR DESCRIPTION
**Pull request by JamesLDeveloper**
Added new Storage Section to Guide.

Have included steps on how to create and access storage.

### CODE REVIEW

File: docs/guide/storage.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+```
+
+</cliWindow>
+
+#### 5. Create a public endpoint URL.
+
+<cliWindow>
+
+```text {1}
+ $ 	gccli fx public
```
Comment (tomaswarynyca) [inline docs/guide/storage.md:147]:
```suggestion
 $ gccli fx public
```

File: docs/guide/storage.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+
+```
+
+#### 4. Deploy your code to GreenCloud
+
+<cliWindow>
+
+```text {1}
+ $  gccli fx deploy
```
Comment (tomaswarynyca) [inline docs/guide/storage.md:131]:
```suggestion
 $ gccli fx deploy
```

File: docs/guide/storage.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+👷 Verifying configuration...
+📦️ Archs: AMD64,ARM64,ARM
```
Comment (tomaswarynyca) [inline docs/guide/storage.md:133]:
```suggestion
📦️ Archs: AMD64
```

File: docs/guide/storage.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+#### 5. Create a public endpoint URL.
+
+<cliWindow>
+
+```text {1}
+ $ 	gccli fx public
+👷 Validating inputs...
+🚨 You must keep the public endpoint safe!
+🔗 Any existing endpoint will be revoked
```
Comment (tomaswarynyca) [inline docs/guide/storage.md:150]:
```suggestion
```

File: docs/guide/storage.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+	// set storage
+	newStorage := base64.StdEncoding.EncodeToString([]byte("{\"admin\":1,\"password\":\"rich\"}"))
+	headers.Set("GC-Storage", newStorage)
+
+	return handler.Response{
+		Body:       []byte(text),
+		StatusCode: http.StatusOK,
+		Header:     headers,
+	}, err
```
Comment (tomaswarynyca) [inline docs/guide/storage.md:120]:
```suggestion
	// set storage
	newStorage := base64.StdEncoding.EncodeToString([]byte("{\"admin\":1,\"password\":\"rich\"}"))
	headers.Set("GC-Storage", newStorage)

	return handler.Response{
		Body:       []byte("I am a storage Test!"),
		StatusCode: http.StatusOK,
		Header:     http.Header{
		"Content-Type": []string{"text/plain; charset=utf-8"},
	    },
	}, err
```

**Review by tomaswarynyca — CHANGES_REQUESTED** [review]
You are showing how to set the storage but never how to get it back.

```
// get storage
storage := req.Header.Get("GC-Storage")
text, _ := base64.StdEncoding.DecodeString(storage)
```

```
// set storage
newStorage := base64.StdEncoding.EncodeToString([]byte({\"user\":\"admin\",\"password\":\"rich\"}"))
headers.Set("GC-Storage", newStorage)
```

**Comment by JamesLDeveloper** [comment]
> You are showing how to set the storage but never how to get it back.
> 
> ```
> // get storage
> storage := req.Header.Get("GC-Storage")
> text, _ := base64.StdEncoding.DecodeString(storage)
> ```
> 
> ```
> // set storage
> newStorage := base64.StdEncoding.EncodeToString([]byte({\"user\":\"admin\",\"password\":\"rich\"}"))
> headers.Set("GC-Storage", newStorage)
> ```

I've commited a change, please can you take another look and see if it is ok now?

**Review by tomaswarynyca — CHANGES_REQUESTED** [review]
Adding examples we've been looking at via Slack

**Comment by JamesLDeveloper** [comment]
Added examples in tabs and changed storage size from 16mb to 1mb. Push is showing both a storage.md and Storage.md. Not sure which will actually be updated in the repo. Thanks

**Review by tomaswarynyca — APPROVED** [review]

**Pull request merged**